cockpit/src/subscriptions-view.jsx

```diff
@@ -148,14 +148,72 @@ class SubscriptionStatus extends React.Component {
         let label;
         let action;
         let note;
+        let syspurpose;
+        let sla;
+        let usage;
+        let role;
+        let add_ons;
+        let syspurpose_status;
         let isUnregistering = (this.props.status === "unregistering");
+        if (this.props.syspurpose["service_level_agreement"]) {
+            sla = (
+                <div>
+                    { _("Service Level:  ") }
+                    { _(String(this.props.syspurpose["service_level_agreement"])) }
+                </div>
+            );
+        }
+        if (this.props.syspurpose["usage_type"]) {
+            usage = (
+                <div>
+                    { _("Usage Type:  ") }
+                    { _(String(this.props.syspurpose["usage_type"])) }
+                </div>
+            );
+        }
+        if (this.props.syspurpose["role"]) {
+            role = (
+                <div>
+                    { _("Role:  ") }
+                    { _(String(this.props.syspurpose["role"])) }
+                </div>
+            );
+        }
+        if (this.props.syspurpose["add-ons"]) {
+            add_ons = (
+                <div>
+                    { _("Add-Ons:  ") }
+                    { _(String(this.props.syspurpose["add-ons"].join(", "))) }
```

Comment:
This assumes that the value of ['add-ons'] is a list/array. This will throw "Exception: TypeError: this.props.syspurpose["add-ons"].join is not a function" (or similar depending on browser) if the value of addons is not a list.